app/helpers/middleware_server_helper/textual_summary.rb

```diff
@@ -62,16 +62,25 @@ def textual_version
   end
 
   def textual_lives_on
-    lives_on_ems = @record.try(:lives_on).try(:ext_management_system)
+    lives_on = @record.try(:lives_on)
+    lives_on_ems = lives_on.try(:ext_management_system)
     return nil if lives_on_ems.nil?
-    lives_on_entity_name = _("Virtual Machine")
+
+    if lives_on.kind_of?(Container)
+      lives_on_entity_name = _("Underlying Container")
+      lives_on_controller_name = 'container'
+    else
+      lives_on_entity_name = _("Underlying Virtual Machine")
+      lives_on_controller_name = 'vm_or_template'
+    end
+
     {
-      :label => "Underlying #{lives_on_entity_name}",
```

Comment:
Why is "underlying" not reused as previously? now it is repeated in line 70 and in line 73